.github/workflows/test.yml

```diff
@@ -163,14 +163,19 @@ jobs:
     - name: Install deps
       # Not using setup-msys2 `install` option to make it easier to copy/paste
       run: |
-        pacman --noconfirm -S --needed git rust
+        pacman --noconfirm -S --needed git rust python3 diffutils tmux
     - name: cargo build
       run: |
         cargo build
-    - name: smoketest
+    - name: cargo test
       # We can't run `build_tools/check.sh` yet, there are just too many failures
       # so this is just a quick check to make sure that fish can swim
```

Comment:
"too many failures" is no longer really accurate,
and for us it's obvious enough that only the main ubuntu job runs all lints that check.sh also runs,
so we could drop the comment now